### DISCUSSION TITLE
how to disable traefik proxy request buffer

### DISCUSSION THREAD

**Discussion by wl2659297** [k3s-io/k3s#8802] [Q&A]
how to disable traefik proxy request buffer in k3s？

just like disable nginx ingress  proxy buffer:   
     nginx.ingress.kubernetes.io/proxy-request-buffering: off

i want to upload big file and show upload processing percent in my web UI. so i need to disable it.

thanks!!!

**Comment by brandond** [comment]
Questions about Traefik are best directed at that project: https://community.traefik.io/
